components/TProfile.vue

```diff
@@ -27,30 +27,40 @@
           :url="$route.fullPath"
           :text="profile.name"
           type="context"
-          label="Share"
+          :label="$t('share.title')"
         />
       </TDropdown>
     </THeader>
 
     <template v-if="uid === profile.id">
       <div v-if="intro.visible" class="m-4 rounded border p-4">
         <h2 class="border-b p-4 font-bold text-lg -m-4 mb-4">
-          Welcome to WeDance!
+          {{ $t('myprofile.intro.visible.title') }}
```

Comment:
Title belongs to visible? Doesn't make much sense ;)

```suggestion
          {{ $t('myprofile.intro.title') }}
```